pandas/tests/frame/test_missing.py

```diff
@@ -983,3 +983,11 @@ def test_interp_time_inplace_axis(self, axis):
         result = expected.interpolate(axis=0, method="time")
         expected.interpolate(axis=0, method="time", inplace=True)
         tm.assert_frame_equal(result, expected)
+
+    def test_interp_string_axis(self):
+        # GH 25190
```

Comment:
can you add a few words here so the reader doesnt need to look up 25190?  very brief is OK